src/pages/html_conversions/new_page.cr

```diff
@@ -7,17 +7,16 @@ class HtmlConversions::NewPage < PageLayout
       h2 title, class: "text-3xl mb-4"
 
       para "Lucky uses Crystal classes and methods to generate HTML. It may sound crazy at first, but the advantages are numerous. Never accidentally print nil to the page, extract and share partials using regular methods. Easily read an entire page by looking at just the render method. Text is automatically escaped for security. And it’s all type safe. That means no more unmatched closing tags, and never rendering a page with missing data."
-
-      form_for HtmlConversions::Create, class: "my-4 space-y-2" do
-        div class: "space-y-1" do
-          h3 "HTML Input", class: "text-2xl"
+      div class: "space-y-1" do
+        h3 "HTML Input", class: "text-2xl"
+        form_for HtmlConversions::Create, data_turbo_frame: "html-conversion", class: "autosave my-4 space-y-2" do
           textarea input, name: "input", class: "appearance-none mt-2 border-2 border-gray-200 rounded w-full py-2 px-4 leading-tight focus:outline-none focus:bg-white", placeholder: "Paste your HTML here", attrs: [:required], rows: 4
         end
+      end
 
-        render_down_arrow
-        render_convert_button
-        render_down_arrow
+      render_down_arrow
 
+      tag "turbo-frame", id: "html-conversion" do
```

Comment:
Is this required for all forms with turbo?

Reply:
No, only forms that redirect/render to themselves

Reply:
It's just a catch for turbo to say "hey I don't know if you want me to morph the page or reload the page"
So you can either add a "target=_top" to the form to force a normal form submit, or you can add a frame to tell it what part of the page to update

Reply:
Ah, ok. So in this case, it would be up to the developer to just "know" that this had to be added? :thinking:  I'm just trying to think from a Lucky standpoint, can we catch this sort of error in development, or would it be a case of just adding documentation around it....